admin-console/src/main/java/com/agiletec/apsadmin/admin/BaseAdminAction.java

```diff
@@ -195,9 +195,13 @@ private void addPages(IPage page, List<IPage> pages) {
 		if (page.getGroup().equals(Group.FREE_GROUP_NAME)) {
 			pages.add(page);
 		}
-		IPage[] children = page.getChildren();
+		boolean isOnline = page.isOnline();
+		String[] children = page.getChildrenCodes();

```

Comment:
Can this ever be null or is a null check required? Since page is an interface seems like this could die here if the page has no children?

Reply:
I'll fix it